sematic/db/migrations/20221208183307_create_server_settings_file.py

```diff
@@ -0,0 +1,21 @@
+# Standard Library
```

Comment:
Please register this script in the migration scripts list in `schema.sql.sqlite`, otherwise it won't be executed.

Reply:
The schema file has no impact on what gets run or not. The migration script checks the user's DB to know what needs to be executed. The schema file is just a snapshot of the most up-to-date state of the DB.